src/PhpSpreadsheet/Writer/Xlsx/Drawing.php

```diff
@@ -490,4 +500,18 @@ public function allDrawings(Spreadsheet $spreadsheet)
 
         return $aDrawings;
     }
+
+    /**
+     * @param XMLWriter $objWriter
+     * @param $pHlinkClickId
+     */
+    private function writeHyperLinkDriwing(XMLWriter &$objWriter, $pHlinkClickId)
```

Comment:
Change `writeHyperLinkDriwing` to `writeHyperLinkDrawing`. And `&$objWriter` to `$objWriter`. And document type of `$pHlinkClickId`